### PR TITLE
fix migrations problem

### LINKED ISSUE #147: We should list duration of analysis
In the analysis list please show the duration of each analysis. It will be useful for debugging slow processing workflows and help to manage user expectations as to how long they should wait for their jobs.

screen shot 2017-06-24 at 10 00 50 pm

### PR DESCRIPTION
**Pull request by boney-bun**
this commit aims to fix the migration problem.

fix #147

### CODE REVIEW

**Comment by lucernae** [comment]
You had to leave the previous migration file and run makemigration after that so the datetime migration is included in the next file (002).

File: migrations/0001_initial.py [hunk 1]
```diff
@@ -18,13 +19,16 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('user_title', models.CharField(help_text=b'Title to assign after analysis is generated.', max_length=255, null=True, verbose_name=b'User defined title for analysis', blank=True)),
+                ('user_extent', models.CharField(help_text=b'User defined BBOX for analysis extent', max_length=255, null=True, verbose_name=b'Analysis extent', blank=True)),
```
Comment (lucernae) [inline migrations/0001_initial.py]: All this changes should be included in the next migration --> 003.
Migration file is like a record that keeps the structure version of database. If you migrate staging like this, django will expect that start_time is already exists in 001, in fact, it doesn't exists in 001 in that current database.

File: migrations/0002_analysis_user_extent.py [hunk 2]
```diff
@@ -1,19 +0,0 @@
-# -*- coding: utf-8 -*-
```
Comment (lucernae) [inline migrations/0002_analysis_user_extent.py]: Do not delete this file, it is a record of database structure version. Always change the new one you want to add --> 003. Deleting migration file only happens when the migration file is squashed.

**Review by lucernae — CHANGES_REQUESTED** [review]
You need to redo migrations. Ping me if you need help.

**Review by lucernae — APPROVED** [review]

**Comment by lucernae** [comment]
LGTM, @boney-bun , thanks!

**Comment by boney-bun** [comment]
thanks @lucernae

**Pull request merged**